rclpy/src/rclpy/_rclpy_pybind11.cpp

```diff
@@ -69,6 +69,8 @@ PYBIND11_MODULE(_rclpy_pybind11, m) {
   .value("RCL_PUBLISHER_LIVELINESS_LOST", RCL_PUBLISHER_LIVELINESS_LOST)
   .value("RCL_PUBLISHER_OFFERED_INCOMPATIBLE_QOS", RCL_PUBLISHER_OFFERED_INCOMPATIBLE_QOS);
 
+  m.attr("rclpy_RMW_DURATION_INFINITE") = py::int_(rmw_time_total_nsec(RMW_DURATION_INFINITE));
```

Comment:
can this be just named `DURATION_INFINITE`?

```suggestion
  m.attr("DURATION_INFINITE") = py::int_(rmw_time_total_nsec(RMW_DURATION_INFINITE));
```

The prefixes are ok for C macros which polute the global namespace but aren't needed here (`_rclpy.DURATION_INFINITE` is clear enough).

Reply:
I agree with this assessment. However, the way that the current pybind code works, anything that doesn't start with `rclpy_` is ignored in the module merge (https://github.com/aws-ros-dev/rclpy/blob/emersonknapp/topic-info-duration-infinite/rclpy/rclpy/impl/implementation_singleton.py#L46), so I had to add an `rclpy_` prefix.

Reply:
@sloretz what do you think?

Reply:
@sloretz friendly ping! should we consider changing the pybind name semantics to expose constants, or keep this as-is?

Reply:
Based on the latest code, I was able to remove the prefix from this constant